consensus/Cargo.toml

```diff
@@ -21,6 +21,8 @@ hex = "0.4"
 lazy_static = "1"
 thiserror = "1"
 
+mina-signer = {git = "https://github.com/willemolding/proof-systems", rev = "c05889b53668b44bb8944737581a24399747fb78"}
```

Comment:
is this gonna merged?

Reply:
Hopefully. We should hold off merging this and see. It isn't ideal to be referencing a fork on my github account